src/views/example/cart-screen/cart-screen.tsx

```diff
@@ -12,7 +12,15 @@ import { BulletItem } from "../bullet-item"
 import { Api } from "../../../services/api"
 import { save } from "../../../lib/storage"
 import { inject, observer } from "mobx-react"
-import { RootStore } from "../../../app/root-store";
+import { RootStore } from "../../../app/root-store"
+import { Mutation } from 'react-apollo';
+import gql from 'graphql-tag'
+import ApolloClient from "apollo-boost";
+export const client = new ApolloClient({
+  uri: "http://localhost:4000/graphql"
+});
```

Comment:
Make sure to remove this part and instead import client from Main.tsx instead

Reply:
Yeah will do once everything is merged